tests/setups.py

```diff
@@ -0,0 +1,92 @@
+import os
```

Comment:
Kind of a nitpick, but can we call this file `setup.py`?

Reply:
You are right, this name is unimaginative. However this is not exactly a "setup" either. How would you like "helpers.py" (only because I can't use test_helpers.py)?

Reply:
I changed it to helpers, but I hadn't seen Blady's comment. However it is not exclusively aws helper so this is more appropriate